docs/extensbility-in-blocks/README.md

```diff
@@ -0,0 +1,73 @@
+---
+category_title: Extensibility in blocks 
+category_slug: extensibility-in-blocks
+post_title:  Extensibility in blocks
+---
+
+# Extensibility in WooCommerce Blocks <!-- omit in toc -->
+
+## Table of Contents <!-- omit in toc -->
```

Comment:
We need to strip table of contents since `#` links don't translate properly from markdown -> the docs site in this way (and one is auto added).